docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,520 @@
+
+<Tabs>
+<TabItem label="TLS Routing">
+
+| Port | Description |
+| - | - |
+| `443` | ALPN port for TLS Routing. |
+
+</TabItem>
+<TabItem label="Separate Ports">
+
+These ports are required:
+
+| Port | Description |
+| - | - |
+| `3023` | SSH port for clients connect to. |
+| `3024` | SSH port used to create reverse SSH tunnels from behind-firewall environments. |
+| `443` | HTTPS connections to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. |
+
+You can leave these ports closed if you are not using their corresponding
+services:
+
+| Port | Description |
+| - | - |
+| `3026` | HTTPS Kubernetes proxy |
+| `3036` | MySQL port |
+| `5432` | Postgres port |
+
+</TabItem>
+</Tabs>
+
+*This is the same table of ports you used to configure the load balancer.*
+
+### License file
+
+If you are deploying Teleport Enterprise, you need to download a license file
+and make it available to your Teleport Auth Service instances.
+
+To obtain your license file, visit the [Teleport customer
+dashboard](https://dashboard.gravitational.com/web/login) and log in. Click
+"DOWNLOAD LICENSE KEY". You will see your current Teleport Enterprise account
+permissions and the option to download your license file:
+
+![License File modal](../../img/enterprise/license.png)
+
+The license file must be available to each Teleport Auth Service instance at
+`/var/lib/teleport/license.pem`. 
+
+### Configuration
+
+Create a configuration file and provide it to each of your Teleport instances at
+`/etc/teleport.yaml`. We will explain the required configuration fields for a
+high-availability Teleport deployment below. These are the minimum requirements,
+and when planning your high-availability deployment, you will want to follow a
+more specific [deployment guide](introduction.mdx) for your environment. 
+
+#### `storage` 
+
+The first configuration section to write is the `storage` section, which
+configures the key-value store and object store for the Teleport Auth Service.
+
+The example configurations below apply to Teleport deployments that use Amazon
+DynamoDB, Google Cloud Firestore, and etcd as the Auth Service backend:
+
+<Tabs>
+<TabItem label="Amazon DynamoDB">
+
+```yaml
+version: v2
+teleport:
+  storage:
+    type: dynamodb
+    region: "us-east-1" # Or another region
+    table_name: "my-kv-table"
+    audit_events_uri: ["dynamodb://my-audit-events-table", "stdout://"]
+    continuous_backups: true
+    audit_sessions_uri: "s3://my-session-recording-bucket"
+```
+</TabItem>
+<TabItem label="Google cloud Firestore">
+```yaml
+version: v2
+teleport:
+  storage:
+    type: firestore
+    project_id: my-google-cloud-project
+    collection_name: my-cluster-state-table
+    credentials_path: /etc/teleport-secrets/gcp-credentials.json
+    audit_events_uri: ['firestore://my-audit-events-table?projectID=my-google-cloud-project&credentialsPath=/etc/teleport-secrets/gcp-credentials.json', 'stdout://']
```

Comment:
It's very important that this is a different collection than the one mentioned in `collection_name`. We cover this in the backends doc page, so I'm concerned that duplicating the information here (in less detail) will cause readers to never see the other page and miss important details.